### ISSUE TITLE
[bug] fails on python3.5.6

### ISSUE THREAD

**Issue by atb00ker** [facebook/pyre-check#102]
Syntax Error:
```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/bin/pyre", line 7, in <module>
    from pyre_check.pyre import main
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/pyre_check/pyre.py", line 14, in <module>
    from . import (
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/pyre_check/commands/__init__.py", line 9, in <module>
    from .initialize import Initialize as Initialize  # noqa
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/pyre_check/commands/initialize.py", line 24
    self._local: bool = arguments.local
               ^
SyntaxError: invalid syntax
```

Steps To Reproduce:
1. Install pyre-check on Python 3.5.6
2. Run any command: ( I did `pyre --version`)

**Comment by dark** [comment]
Indeed, thanks for the report. Fix incoming.

**Comment by dark** [comment]
First fix at 0988e2f9b1e36dbbed07ae7a03f616d6f0233a13, another one will follow.
